cmd/utils/utils.go

```diff
@@ -26,11 +26,13 @@ func InitGoMod(projectName string, appDir string) error {
 	return nil
 }
 
-func GoGetPackage(appDir, packageName string) error {
-	if err := ExecuteCmd("go",
-		[]string{"get", "-u", packageName},
-		appDir); err != nil {
-		return err
+func GoGetPackage(appDir string, packages []string) error {
+	for _, packageName := range packages {
+		if err := ExecuteCmd("go",
+			[]string{"get", "-u", packageName},
+			appDir); err != nil {
+			return err
+		}
```

Comment:
had to change this function to support the echo middleware lib. I assumed we didn't have a way to get more than one package per framework @Melkeydev

Reply:
@xsadia You actually dont need to explicitly do a go get for middleware.
```
go get github.com/labstack/echo/v4
```

and then just having the imports as:
```
import (
  "github.com/labstack/echo/v4"
  "github.com/labstack/echo/v4/middleware"
  "net/http"
)
```
Should work

Reply:
When I create a project without explicitly doing a go get for the middleware I get 

```
error while importing github.com/labstack/echo/v4/middleware: missing go.sum entry for module providing package github.com/golang-jwt/jwt (imported by github.com/labstack/echo/v4/middleware); to add:
go get github.com/labstack/echo/v4/middleware@v4.11.2
```
I don't know if I missed something @Melkeydev

Reply:
:thinking: Oh interesting - the Echo read me is a little misleading then. 
NVM my previous comment. I am going to pull this PR down and check it out later today (looking good though)

Reply:
Ok! Just ping me if you need anything